### PR TITLE
FF123 - CredentialContainer.get .create support cross-origin calls

### PR DESCRIPTION
**Pull request by hamishwillee**
FF123 supports Feature policy [`publickey-credentials-create`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy/publickey-credentials-create) in https://bugzilla.mozilla.org/show_bug.cgi?id=1870863
That is updated in #22172

But the reason we add this is to enable calling `navigator.credentials.create({publicKey})` (for the Web Authentication API) in a cross-origin iframes.
So what this PR does is note that support in the `create()` method where you are likely to see it. The version uses is the earliest version of `publickey-credentials-create` (ignoring particular policy used to implement it).

Similarly in 118 we added support for the `get()` call with the `publickey-credentials-get` feature. So I added that too.

I did this separate to #22172 because I think this should be here, and done in this way, but I didn't want to block the unambiguously correct entry.

Related docs work can be tracked in https://github.com/mdn/content/issues/31890

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks @Elchi3 - fixed. Sometimes `npm run fix` goes bonkers, but still should have caught that.

**Comment by hamishwillee** [comment]
> Normally I think entries like this aren't needed. The docs should document the behavior and the compat info for publickey-credentials-create should be linked to from the .get and .create methods.
> However, if you feel strongly about including this here, I will not argue about it :)

@Elchi3 Thanks for review - choosing to close.

TLDR;
1. FYI only, the permission only allows cross-origin for get requests using the Web authentication API, which is why I mentioned that. So both text and key names would have been wrong/insufficient (both my versions and yours).
2. The docs do cross link as you have suggested, so the information is accessible.
3. I think I'd find the information useful here, but not enough to overcome the normal ways of working.
4. Thanks again!
